src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString
+    .split(';')
+    .map((style) => {
+      return style.trim();
+    })
+    .filter((style) => style !== '');
```

Comment:
The chained methods .split, .map, and .filter should be indented at the same level to improve readability.